Applied changes to the CloudServices section in web ui.

MMaiero opened the pull request:
- Add some instruction text at the top of the page
- Move Refresh Button to last
- In the new Cloud Service dialog, remove the icons from the buttons and just have
them as Submit and Cancel

- In CloudService updated the following property descriptions:
  - device.display.name;
  - topic.control-prefix;
  - enable.default.subscriptions.
- In MQTTDataTransport:
  - topic.context.account-name.

Signed-off-by: MMaiero <matteo.maiero@eurotech.com>

#1268

pierantoniomerlino commented on kura/org.eclipse.kura.core.cloud/OSGI-INF/metatype/org.eclipse.kura.cloud.CloudService.xml:
```diff
@@ -83,7 +83,7 @@
             default="true"
-            description="By disabling default subscriptions the gateway will not be remotely manageable"/>
+            description="Disables the default subscriptions to the gateway management MQTT topics. When disabled, the gateway will not be remotely manageable."/>
             
```
Why "Disables the ..."? It is better to use "Enables the..." since the property name is **enable**.default.subscriptions.

MMaiero replied:
https://github.com/eclipse/kura/issues/1268

pierantoniomerlino replied:
👍

MMaiero committed:
Applied changes to the CloudServices section in web ui.

- Add some instruction text at the top of the page
- Move Refresh Button to last
- In the new Cloud Service dialog, remove the icons from the buttons and just have
  them as Submit and Cancel

- In CloudService updated the following property descriptions:
  - device.display.name;
  - topic.control-prefix;
  - enable.default.subscriptions.
- In MQTTDataTransport:
  - topic.context.account-name.

Signed-off-by: MMaiero <matteo.maiero@eurotech.com>

MMaiero commented:
Is the PR approved and ready to merge?
